cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1388,28 +1388,28 @@ def test_advanced_problems(self):
         Test the handling of advanced problem templates.
         """
         problem_templates = self.get_templates_of_type('problem')
-        ora_template = self.get_template(problem_templates, u'Peer Assessment')
-        self.assertIsNotNone(ora_template)
-        self.assertEqual(ora_template.get('category'), 'openassessment')
-        self.assertIsNone(ora_template.get('boilerplate_name', None))
+        circuit_template = self.get_template(problem_templates, u'Circuit Schematic Builder')
```

Comment:
Nit: I'd prefer to test for a more "general" template here, as I'd eventually prefer to see the Circuit Schematic Builder also removed from the core platform; it's very edX specific.